app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Note for methods which have no view (update, destroy, create, complete), you don't need instance variables.  Local variables will do the job.